src/util/MemorySize/MemorySize.cpp

```diff
@@ -63,7 +58,39 @@ std::string MemorySize::asString() const {
 
 // _____________________________________________________________________________
 MemorySize MemorySize::parse(std::string_view str) {
-  return MemorySizeParser::parseMemorySize(str);
+  if (auto matcher =
+          ctre::match<"(\\d+(?:\\.\\d+)?) ([kKmMgGtT][bB]?|[bB])">(str)) {
+    auto amountString = matcher.get<1>().to_view();
+    double amount;
+    std::from_chars(amountString.begin(), amountString.end(), amount);
```

Comment:
I think not all of our compilers support that, but there's `absl::from_chars` for that reason.